packages/core-bridge/common.d.ts

```diff
@@ -0,0 +1,7 @@
+export const targets: string[];
```

Comment:
We could move the `common.js` and `index.js` to TS, but decided to defer to another PR unless there's opposition